### DISCUSSION TITLE
docs/micro-service/scaffold

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#22] [General]
# docs/micro-service/scaffold

GoFrame 框架包含多个微服务组件，并提供了易用的 GRPC 脚手架模块和工具。脚手架由 grpcx 社区包实现： https://github.com/gogf/gf/tree/master/contrib/rpc/grpcx 包含多个模块。

https://goframe.org/docs/micro-service/scaffold

**Comment by lazydo9** [comment]
这里为什么会调两遍一模一样的DEBUG日志？是什么配置项吗？
